test/e2e/validators.go

```diff
@@ -196,7 +195,7 @@ func validateBuildRunToSucceed(
 				return ""
 			}
 			return taskRun.Status.Conditions[0].Reason
-		}, 300*time.Second, 5*time.Second).Should(BeElementOf(pendingAndRunningStatues), "TaskRun not pending or running")
+		}, 300*time.Second, 5*time.Second).Should(BeElementOf(pendingAndRunningStatues), "TaskRun REASON not pending or running")
```

Comment:
Nice check.

Just one comment: should this sentence be: TaskRun REASON **is** not pending or running ?

Reply:
@zhangtbj done